app/javascript/angular/code/controllers/_sessions_list_ctrl.js

```diff
@@ -26,28 +26,24 @@ export const SessionsListCtrl = (
     $window.sessions = sessions = $scope.sessions;
     $scope.sessionsForList = [];
 
-    // prolly this can be removed
-    if (_(params.get("selectedSessionIds", [])).isEmpty()) {
-      params.update({ selectedSessionIds: [] });
-    }
-
-    sessions.reSelectAllSessions();
+    if (sessionsUtils.isSessionSelected())
```

Comment:
Is this `if` needed? Cause it seems it wasn't there before

Reply:
Yes.
Before there was a method `reSelectAllSessions` which had a loop on all selected sessions. So if there was no session selected it didn't execute the code in the loop. Now there are only two possibilities - there is a selected session or not. So it doesn't make sense to loop over a list of selected session - easier to just have this `if`.